pkg/types/visibility.go

```diff
@@ -33,11 +33,17 @@ type Visibilities struct {
 
 // Visibility struct
 type Visibility struct {
-	ID            string    `json:"id"`
-	PlatformID    string    `json:"platform_id"`
-	ServicePlanID string    `json:"service_plan_id"`
-	CreatedAt     time.Time `json:"created_at"`
-	UpdatedAt     time.Time `json:"updated_at"`
+	ID            string             `json:"id"`
+	PlatformID    string             `json:"platform_id"`
+	ServicePlanID string             `json:"service_plan_id"`
+	CreatedAt     time.Time          `json:"created_at"`
+	UpdatedAt     time.Time          `json:"updated_at"`
+	Labels        []*VisibilityLabel `json:"labels,omitempty"`
```

Comment:
would it make sense for this to be a map instead ? since we have uniqueness on keys and possibly , when one reads the visibilities with labels from db and if the purpose is processing something based on labels , you would probably be interested in particular label (labelkey)

Reply:
something like `type Labels map[string][]string`? Marshalling and unmarshalling would be a hassle then probably, because you'd have to define a helper struct.